locale/en_US/manager.xml

```diff
@@ -477,6 +477,7 @@
 	<message key="plugins.importexport.common.error.unknownEncoding">Unknown encoding {$param}</message>
 	<message key="plugins.importexport.common.error.encodingError">The content is not encoded as {$param}</message>
 	<message key="plugins.importexport.common.error.unknownUserGroup">Unknown user group {$param}</message>
+	<message key="plugins.importexport.common.error.missingGivenName">The given name in the submission locale is required for the author {$param}</message>
```

Comment:
> The author {$authorName} does not have a Given Name in the submission's language, {$localeName}. The submission can not be exported without this information.

Reply:
Currently we only can provide one $param variable :-\ Would you maybe have any other suggestion? THANKS!!!

Reply:
Oh, do you mean in the import/export code specifically, the error message reporting is done in a way that only allows one param?

I suppose the author's name is probably more important. Let's try:

> The author {$authorName} does not have a Given Name in the submission's language. The submission can not be exported without this information.

Reply:
Ah, I think you are right -- I mistaken it to be like FormValidators... Hmmm... OK, then I will do that... THANKS!!!

Reply:
Could I use locale like en_US for that $localeName? -- I do not have to use a function that will return me the name and it is maybe directly clear what attribute is missing... ?
And, should Given Name be with capital letters? :-)
THANKS!!!

Reply:
No, either use the full locale name, or just leave it out. I don't think non-technical people know what `en_US` means.

Yeah, on second thought let's not capitalize given name.

Reply:
OK, I will then try with the locale full name.
And I just changed "exported" with "imported" -- it is an error message when the import XML misses the given name in that locale...